pkg/reconciler/route/resources/ingress.go

```diff
@@ -136,6 +136,11 @@ func makeIngressSpec(
 	rules := make([]netv1alpha1.IngressRule, 0, len(names))
 
 	featuresConfig := config.FromContextOrDefaults(ctx).Features
+	networkConfig := config.FromContextOrDefaults(ctx).Network
+	internalTLS := false
```

Comment:
I'm not sure if we'd end up wanting to pass along the `ActivatorCA` so that backends could validate it.